core/services/offchainreporting2/validate.go

```diff
@@ -39,9 +39,6 @@ func ValidatedOracleSpecToml(config Config, tomlString string) (job.Job, error)
 	if jb.Type != job.OffchainReporting2 {
 		return jb, errors.Errorf("the only supported type is currently 'offchainreporting2', got %s", jb.Type)
 	}
-	if !tree.Has("isBootstrapPeer") {
```

Comment:
Probably we want to error here now right? If they try to submit the old style of job spec we wanna reject it with an error